lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -311,13 +311,17 @@ describe('ReportRenderer', () => {
           'link-in-text-block': {enabled: false},
           // Report has empty links prior to i18n-ing.
           'link-name': {enabled: false},
+          // TODO .lh-category-wrapper:nth-child(2) > .lh-category > .lh-clump--manual.lh-clump.lh-audit-group > summary
+          'nested-interactive': {enabled: false},
```

Comment:
I guess this is the selector of the failure? This is going to be lost down here. How hard is the fix?

Reply:
I don't know, it was Friday evening, this PR got big, and I just wanted to expedite feedback (since I knew there'd be quite a bit).

don't plan on pushing this PR w/o looking into this more. looks fun.

Reply:
https://dequeuniversity.com/rules/axe/4.2/nested-interactive

so it seems that this rule fails for any `details>summary` with so much as an anchor. Seems rather restrictive, possible oversight in the axe rule?

Reply:
I think this rule is not ready for LH–should we hide it, keep it to zero weight, or delete?

Reply:
> I think this rule is not ready for LH–should we hide it, keep it to zero weight, or delete?

I vote we delete it and `aria-text` but leave a TODO for `nested-interactive` so we update it next time if ready